lib/order.rb

```diff
@@ -0,0 +1,39 @@
+# As a customer
+# So that I am reassured that my order will be delivered on time
+# I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
+
+require_relative 'menu'
+require_relative 'message'
+
+class Order
+
+  attr_reader :current_order
+
+  def initialize
+    @menu = Menu.new
```

Comment:
Your order class is tightly coupled with your Menu class. Could we use dependency injection here?

https://github.com/makersacademy/skills-workshops/blob/f5b4801840fe07d26ff70341652dc81dcda12289/practicals/object_oriented_design/dependency_injection.md might be a helpful resource